include/RAJA/policy/sequential/params/reduce.hpp

```diff
@@ -7,6 +7,50 @@ namespace RAJA {
 namespace expt {
 namespace detail {
 
+  // Init
+  template<typename EXEC_POL, template <typename, typename, typename> class OP, typename T>
+  camp::concepts::enable_if< std::is_same< EXEC_POL, RAJA::seq_exec> >
+  init(Reducer<OP<ValOp<ValLoc<T>,OP>, ValOp<ValLoc<T>,OP>, ValOp<ValLoc<T>,OP>>, ValOp<ValLoc<T>,OP>>& red) {
+    red.val = OP<ValOp<ValLoc<T>,OP>, ValOp<ValLoc<T>,OP>, ValOp<ValLoc<T>,OP>>::identity();
```

Comment:
We should figure out why using `Op<ValOp<ValLoc<T>,OP>, ...>::identity()` works here but `Op<ValOp<T,OP>, ...>::identity()` doesn't work below.